cmake/Buildbipedal-locomotion-framework.cmake

```
@@ -0,0 +1,33 @@
+# Copyright (C) 2020  iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Giulio Romualdi <gulio.romualdi@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+
+set(bipedal-locomotion-framework_DEPENDS "")
```

Probably in the CMake world we can stick to the CMake package name for consistency, so:
```suggestion
set(BipedalLocomotionFramework_DEPENDS "")
```